src/qusbmoded.cpp

```diff
@@ -399,7 +407,11 @@ void QUsbModed::updateSupportedModes(QString aModes)
 
 void QUsbModed::updateAvailableModes(const QString &aModes)
 {
+#if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
     const QStringList result = aModes.split(',', QString::SkipEmptyParts);
+#else
+    const QStringList result = aModes.split(',', Qt::SkipEmptyParts);
+#endif
```

Comment:
I'd rather check versions with qt 5.14 which introduced the new api. Doesn't need qt 6.